components/dropdown/dropdown-content-mixin.js

```diff
@@ -978,8 +1011,60 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		return false;
 	}
 
-	_getPosition(spaceAround, targetWidth, contentWidth) {
-		const centerDelta = contentWidth - targetWidth;
+	_getPointerPosition(targetRect) {
```

Comment:
This is completely new, since with relative positioning, calculating the position of the pointer was simply CSS.